Classes/Bookmark/BookmarkViewController.swift

```diff
@@ -194,4 +199,10 @@ BookmarkHeaderSectionControllerDelegate {
         cloudStore.clear()
     }
 
+    // MARK: BookmarkSectionControllerDelegate
+
+    func didSwipeToDelete(at indexPath: IndexPath) {
+        let graphQLID = cloudStore.ids[indexPath.section-1]
```

Comment:
Hm, is this guaranteed to always be the correct `id`? I'm a bit worried what would happen during a merge or any other change to the store.

Could we undo the delete when the removal would fail? cc @heshamsalman

Reply:
@BasThomas 🤔 

Initial thought is that there could be edge-cases here where we end up deleting the wrong bookmark, but it's _probably_ fine. 

My only concern is the scenario where a sync w changes triggered by another devices occurs _just before_ this method fires.

Reply:
@BasThomas While playing around with this, I observed that the selected `id` was consistently correct. I'm not familiar with the migration flow, but what I see is during a merge, the new `id`'s are appended onto the `storage` which I don't believe would effect the deletion unless the deleted `id` was the same? 🤔 

@heshamsalman That's a great point. I'm not sure how this would be remedied? maybe there's some state we can check?

Reply:
What about verifying with something other than the `id`? Not sure if that's trivial / worth it.

Otherwise we'll ship it like this for now, your call @heshamsalman 😊

Reply:
I can display an alert that shows the repo/issue name for the user to confirm? The tradeoff here is that the user would need to go through an additional step to delete. Not sure how that'll be UX wise?

Here's a gif of that change:
![ScreenRecording 2019-05-31 at 10 25 01 AM](https://user-images.githubusercontent.com/7361784/58712425-8024c980-838e-11e9-8f3e-0f1e7c672948.gif)

Reply:
The UX on that isn't ideal, but I think there could be a case made for it. 
I'm for merging this as-is; the likelihood of a storage conflict feels very low to me. 

Thoughts @BasThomas?

Reply:
Well the confirmation doesn't really do anything to prevent the potential merge conflicts, does it?